target/linux/kirkwood/image/Makefile

```diff
@@ -7,9 +7,10 @@ include $(INCLUDE_DIR)/image.mk
 
 KERNEL_LOADADDR:=0x8000
 
-define Device/dsa-migration
-  DEVICE_COMPAT_VERSION := 1.1
-  DEVICE_COMPAT_MESSAGE := Config cannot be migrated from swconfig to DSA
+define Device/kernel-size-migration
+  DEVICE_COMPAT_VERSION := 2.0
+  DEVICE_COMPAT_MESSAGE := Partition design has changed compared to older versions (up to 21.02) due to kernel size restrictions. \
```

Comment:
This should be wrapped to 74 chars. max. Append-metadata will convert leading tabs back to spaces anyway ...

Reply:
I will fix it.
But if #4311 will be merged before 21.02.0, text could be changed, because it will be easier way than revert to stock ( I mean `sysupgrade -n -F factory.bin`).

Reply:
Done